### DISCUSSION TITLE
Failed to load resource error 404

### DISCUSSION THREAD

**Discussion by martinvruiz** [community/community#129533] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, i have a proyect and when i go to github pages this didin't work.

the error is Failed to load resource: the server responded with a status of 404 ()

i alredy check the routes and everything. I don't know how to solve it

the link is https://github.com/martinvruiz/tasktodo

thx

**Comment by davevad93** [comment, marked as answer]
Hi @martinvruiz ,

Make the following changes to your `index.html`:

```html
<!-- Delete this -->
<base href="/">

<!-- Delete the "/" in the CSS and JS path -->
<link rel="stylesheet" href="CSS/tailwind.css">
<script src="JS/main.js"></script>
```

Result:
![00](https://github.com/community/community/assets/94308234/68310da1-92ce-47dd-8e1b-1c6a6eca0fc3)

**Reply by martinvruiz** [reply]
thx. it worked

**Reply by davevad93** [reply]
You're welcome! 👍
